demos/serverless/deploy.js

```diff
@@ -235,7 +255,7 @@ if (!fs.existsSync('build')) {
   fs.mkdirSync('build');
 }
 
-console.log(`Using region ${region}, bucket ${bucket}, stack ${stack}, endpoint ${chimeEndpoint}, enable-termination-protection ${enableTerminationProtection}, disable-printing-logs ${disablePrintingLogs}`);
+console.log(`Using region ${region}, bucket ${bucket}, stack ${stack}, endpoint ${chimeEndpoint}, ChimeMeetingsIADEndpoint=${chimeMeetingsIADEndpoint}, ChimeMeetingsPDXEndpoint=${chimeMeetingsPDXEndpoint}, ChimeMeetingsFRAEndpoint=${chimeMeetingsFRAEndpoint}, ChimeMeetingsSINEndpoint=${chimeMeetingsSINEndpoint}, enable-termination-protection ${enableTerminationProtection}, disable-printing-logs ${disablePrintingLogs}`);
```

Comment:
The same applies here: pass in single argument, not argument per region.

Reply:
We need all of them as we need to switch between them based upon url parameter `controlRegion`.